### DISCUSSION TITLE
What client version am I using?

### DISCUSSION THREAD

**Discussion by cmath52** [fedorovvl/tso_client#410] [General]
please use standard naming rules to have the file version part of the name and add the version info to the file details.

**Comment by skelgaard** [comment]
![billede](https://github.com/user-attachments/assets/526cbc3c-5e5d-43be-b972-ca46e54a6da2)

**Reply by cmath52** [reply]
that is when the client is open in the game window. only...
the actual file has never had version detail it has always been version 1 no mater what version it actually is. It is counterproductive for a user to have to run the file to know what version it is. We should be able to look at file details or the name of the file and know the version.

**Comment by Opferlamm** [comment]
1.5.8.0-5b53cd1
You can use the latest build and see the version number

https://prnt.sc/lPt6s7oMc-CK

**Comment by fedorovvl** [comment]
pls check file version on latest build
https://github.com/fedorovvl/tso_client/commit/6816feefc187192220ecb694fb4241252b289f3c

**Comment by Cdrwbw** [comment]
I think that @cmath52  speak to see version number here
![image](https://github.com/user-attachments/assets/11a9b2c1-355f-4f12-863d-6ac85e15ad38)

**Comment by BasilMas** [comment]
it is ok now
![image](https://github.com/user-attachments/assets/83673901-77e2-435c-b675-996db29f9fe6)
